cmd/kube-proxy/app/server.go

```diff
@@ -146,6 +152,13 @@ func (s *ProxyServer) Run(_ []string) error {
 		}, 5*time.Second)
 	}
 
+	go func() {
+		err := proxy.ServeDNS(dnsHandler)
```

Comment:
SkyDNS spawns goroutines in `Run()` -- https://github.com/skynetservices/skydns/blob/master/server/server.go#L149